commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/PurePursuitControllers.scala

```diff
@@ -166,14 +133,15 @@ trait PurePursuitControllers extends UnicycleCoreControllers {
     secondLookAheadPoint.getOrElse(
       firstLookAheadPoint.getOrElse {
         println(s"expanding look ahead distance to ${1.1 * lookAheadDistance}")
```

Comment:
Do we want to be printing this? If this is going to be called often it could cause serious problems at runtime because `println` is blocking.